packages/server/src/jsRunner/tests/jsRunner.spec.ts

```diff
@@ -50,7 +50,7 @@ describe("jsRunner (using isolated-vm)", () => {
   describe("helpers", () => {
     runJsHelpersTests({
       funcWrap: (func: any) => config.doInContext(config.getAppId(), func),
-      testsToSkip: ["random", "uuid"],
+      testsToSkip: ["random", "uuid", "lorem"],
```

Comment:
Why this? This should work automatically based on the test example

Reply:
Sorry - by bad - I left this in after some fiddling!